tests/scancode/test_cli.py

```diff
@@ -126,15 +134,6 @@ def test_license_option_detects_licenses():
     assert len(open(result_file).read()) > 10
 
 
-def test_can_call_run_scan_as_a_function():
-    from scancode.cli import run_scan
-    test_dir = test_env.get_test_loc('license', copy=True)
-    rc, results = run_scan(test_dir, license=True, copyright=True, return_results=True)
-    assert rc
-    assert len(results['files']) == 2
-    assert not results['headers'][0]['errors']
-
-
```

Comment:
Why di you remove this test? it is quite important to keep. Please add it back

Reply:
When merging I think I accidentally deleted it. sorry.